sunbeam/schedule.cpp

```diff
@@ -53,6 +56,26 @@ namespace {
 
 void sunbeam::export_Schedule() {
 
+static const std::string gtstr = R"(
+Function to identify parent and children
```

Comment:
`Function to get the group topology at a given time.` - it is important that the input argument is a time, the variable should be called e.g. `report_step` - not `arg1`.

Reply:
We could also add the option of giving `time` as a `report_step` or as a `datetime`.

Reply:
Yes, I will try fix that, and also make the ´schedule.group_tree´ return a python dict to provide a better inteface for the user